sourcetracker/_cli/gibbs.py

```diff
@@ -190,21 +191,25 @@ def gibbs(table_fp, mapping_fp, output_dir, loo, jobs, alpha1, alpha2, beta,
         else:
             csources = subsample_dataframe(csources, source_rarefaction_depth)
 
-    # Prepare sink data and rarify if requested.
-    sinks = feature_table.loc[sink_samples, :]
-    if sink_rarefaction_depth > 0:
-        d = (sinks.sum(1) >= sink_rarefaction_depth)
-        if not d.all():
-            too_shallow = (~d).sum()
-            shallowest = sinks.sum(1).min()
-            raise ValueError(('You requested rarefaction of sink samples at '
-                              '%s, but there are %s sink samples that have '
-                              'less sequences than that. The shallowest of '
-                              'these is %s sequences.') %
-                             (sink_rarefaction_depth, too_shallow,
-                              shallowest))
-        else:
-            sinks = subsample_dataframe(sinks, sink_rarefaction_depth)
+    # Prepare to rarify sink data if we are not doing LOO. If we are doing loo,
+    # we skip the rarefaction, and set sinks to `None`.
+    if not loo:
+        sinks = feature_table.loc[sink_samples, :]
+        if sink_rarefaction_depth > 0:
+            d = (sinks.sum(1) >= sink_rarefaction_depth)
+            if not d.all():
+                too_shallow = (~d).sum()
```

Comment:
Could you rename this ``count_too_shallow`` for clarity? This sounds like it should have a boolean value.